css/index.css

```diff
@@ -1,3 +1,262 @@
+}
+
+header nav {
+    margin-top: 40px; 
+    width: 610px;
+    justify-content: space-between;
+    display: flex;
+}
+
+header nav a {
+    text-decoration: none;
+    color: #b0b0b0;
+}
+
+header nav a:hover {
+    text-decoration: underline;
+    color: black;
+}
+
+header .logo {
+    margin: 28px 0 0 91px;
+}
+
+/* Call To Action (CTA) Styles */
+
+.cta {
+    display: flex;
+    margin-top: 72px;
+    justify-content: flex-end;
+}
+
+.cta .cta-text {
+    font-size: 72px;
+    text-align: center;
+    letter-spacing: 5px;
+    display: flex;
+    justify-content: center;
+    flex-flow: column;
+    align-items: center;
+    margin: 0 82px 0 0;
+}
+
+.cta .cta-text button {
+    border: 1px double black;
+    font-size: 16px;
+    background: white;
+    width: 170px;
+    height: 32px;
+    margin-top: 10px;
+    cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+    background: black;
+    color: white;
+}
+
+/* Main Content Styles */
+
+.main-content {
+   border-top: 2px solid black; 
+   border-bottom: 2px solid black;
+   margin: 75px 0 35px 0;
+   padding: 35px 0;
+   font-size: 16px;
+   line-height: 18px;
+}
+
+.main-content .top-content {
+    display: flex;
+}
+
+.main-content .middle-img {
+    display: block;
+    margin: 30px auto;
+}
+
+.main-content .bottom-content {
+    display: flex;
+}
+
+.main-content .bottom-content p {
+    padding-right: 20px;
+}
+
+/* Contact Styles */
+
+.contact p {
+    margin: 15px 0;
+}
+
+/* Footer Styles */
+
+footer {
+    text-align: center;
+    margin: 40px 0 20px;
+}
+
+/* Services Styles */
+
+.services-content {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    line-height: 1.3;
+}
+
+.services-content .services-header-img {
+    margin: 20px 0;
+}
+
+.services-content h1 {
+    font-size: 26px;
+    margin: 10px 0;
+}
+
+.services-content .services-text {
+ border-bottom: 1px solid black;
+ padding-bottom: 40px;
+ margin-bottom: 20px
+}
+
+.services-content .card {
+    padding: 20px;
+    border: 1px solid black;
+    width: 420px;
+    background: whitesmoke;
+    margin: 20px 0;
+}
+
+.services-content .card button {
+    padding: 10px 40px;
+    margin-top: 20px;
+    cursor: pointer;
+    background: white;
+    border-radius: 5px;
+    border: 1px solid black;
+}
+
+.services-content .card button:hover {
+    background: black;
+    color: white;
+} 
+
+.services-content .services-info ul {
+    margin: 20px;
+}
+
+.services-content .services-info ul li{
+    list-style: square;
+    padding: 5px;
+}
+
+.services-content .services-info {
+    display: flex;
+    flex-wrap: wrap;
+    border-top: 2px solid black; 
+    margin: 20px 0 35px 0;
+    padding-top: 40px; 
+}
+
+.services-content .services-info .facts {
+  width: 400px;
+  margin: 0 60px 20px 0;
+}
+
+.services-content .services-info .facts p {
+  margin-bottom: 20px;
+}
+
+.services-content .services-info .services-info-img {
+  width: 420px;
+  height: 300px;
+}
 
```

Comment:
it is nicer to leave your @media queries as the bottom for regular css sheets.

Reply:
/* Tablet */
@media (max-width: 800px) {

    .container {
        width: 100%;
    }

    header {
        flex-wrap: wrap;
    }
  
    header nav {
        width: 768px;
        padding: 0 30px;
    }

    header .logo {
        order: -1;
        display: block;
        margin: 30px auto 0;
    }

    .cta {
        justify-content: center;
    }
    
    .cta .cta-text {
        margin: 0;
    }

    .cta img {
        display: none;
    }

    .main-content .middle-img {
       width: 100%;
    }

    .main-content {
       margin: 75px 20px 35px;
    }

    .text-content {
        margin: 0 10px;
    }

    .contact {
        margin: 0 30px;
    }
}

/* Phone */
@media (max-width: 500px) {

    header {
        justify-content: center;
    }

    header nav {
        width: 100%;
        flex-direction: column;
        margin: 0;
        padding: 0;
    }

    header nav a {
        width: 100%;
        display: block;
        text-align: center;
        padding: 20px 0;
        font-size: 20px;
        border-bottom: 1px solid gray;
    }

    header nav a:first-child {
        border-top: 1px solid gray;
    }

    header nav a:hover {
        background: gray;
    }

    header .logo {
        margin: 20px 0;
    }

    .cta {
        margin-top: 50px;
        flex-flow: column;
        align-items: center;
    }

    .cta img {
        max-width: 80%;
    }

    .main-content .top-content {
        justify-content: space-around;
        flex-direction: column;
        height: 300px;
    }

    .main-content .bottom-content {
        flex-wrap: wrap;
        align-content: space-between;
        height: 440px;
    }

    .text-content {
        margin: 0;
    }

    .contact {
        margin: 20px;
    }
}